### PR TITLE
Implemented openstack instance reboot and fixed terminate validation bug

### PR DESCRIPTION
**Pull request by drmaas**

### CODE REVIEW

File: clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/validators/instance/RebootOpenstackInstancesDescriptionValidator.groovy [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+class RebootOpenstackInstancesDescriptionValidator extends DescriptionValidator<OpenstackInstancesDescription> {
+
+  @Autowired
+  AccountCredentialsProvider accountCredentialsProvider
+
+  @Override
+  void validate(List priorDescriptions, OpenstackInstancesDescription description, Errors errors) {
+    def validator = new OpenstackAttributeValidator("rebootOpenstackInstancesAtomicOperationDescription", errors)
+
+    //validate credentials
```
Comment (duftler) [inline clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/deploy/validators/instance/RebootOpenstackInstancesDescriptionValidator.groovy]: This comment and the next don't really seem to add information. No need to include them in that case.

**Comment by duftler** [comment]
LGTM after addressing the minor comment. I'd lean towards striking those same comments from the other validator as well.

**Comment by derekolk** [comment]
Cool ... I will remove those comments in both validators.  Give me a second to address :)

**Comment by duftler** [comment]
LGTM.

**Pull request merged**